### ISSUE TITLE
RoBERTa support

### ISSUE THREAD

**Issue by sleepinyourhat** [huggingface/transformers#829]
https://twitter.com/sleepinyourhat/status/1151940994688016384

The code/parameters aren't out yet, but I figure it couldn't hurt to put in an obnoxious feature request now!

**Comment by myleott** [comment]
Working on the code/paper release as we speak :) It largely follows the existing masked_lm implementation in fairseq. Happy to help get this integrated here.

**Comment by stefan-it** [comment]
Hi @myleott great news :) I'm really excited about the release 🤗 I've some questions: do you plan to perform any comparisons between RoBERTa and BERT on NER (CoNLL-2003)? 

I've read the [Cloze-driven Pretraining of Self-attention Networks](https://arxiv.org/abs/1903.07785) paper, and if I recall correctly, the implementation is currently done in the `bi_trans_lm` branch in `fairseq`, but do you have any updates on that? It would be awesome if a pre-trained CNN model from that paper could also be integrated into `pytorch-transformers` 😍

**Comment by thomwolf** [comment]
Sounds great @myleott. Keep up updated about the release!

**Comment by myleott** [comment]
Models and README are uploaded: https://github.com/pytorch/fairseq/tree/master/examples/roberta. We submitted the paper to arXiv today so it should be out Sunday evening.

> I've some questions: do you plan to perform any comparisons between RoBERTa and BERT on NER (CoNLL-2003)?

We haven't yet, but it would be interesting to explore. RoBERTa was trained on considerably more data than BERT, so I expect it would do well on NER tasks.

**Comment by julien-c** [comment]
Paper is [out](https://arxiv.org/abs/1907.11692). Thanks @myleott!

**Comment by julien-c** [comment]
Work in progress in #964 feel free to chime in :)

**Comment by pwolff** [comment]
An example showing how RobertaForMaskedLM can be used to predict one or more masked tokens would be wonderful. :)

**Comment by thomwolf** [comment]
Hi @pwolff, at first glance it looks ok to me. You don't need to send the `masked_lm_labels` if you don't use the loss though.

**Comment by songtaoshi** [comment]
@thomwolf hello, I trained the robert on my customized corpus following the fairseq instruction. I am confused how to generate the robert vocab.json and also merge.txt because I want to use the pytorch-transformer RoBERTaTokenizer.

**Comment by songtaoshi** [comment]
@stefan-it  hello, I trained the robert on my customized corpus following the fairseq instruction. I am confused how to generate the robert vocab.json and also merge.txt because I want to use the pytorch-transformer RoBERTaTokenizer.

**Comment by stefan-it** [comment]
@songtaoshi I think this can be done via `subword-nmt`, see this note:

https://github.com/pytorch/fairseq/issues/1163#issuecomment-534098220

**Comment by iedmrc** [comment]
is this still an open issue?

**Comment by julien-c** [comment]
Nope, RoBERTa support was shipped in [v1.1.0](https://github.com/huggingface/transformers/releases/tag/1.1.0)

Thanks all!
